modules/coralenv.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+function coralenv {
+  param=$1
+
+  if [ ! -d /usr/share/doc/python3-coral-enviro ] ; then
+    warn "Error: the Coral python environment is not installed"
+    while true; do
+      read -n 1 -pr "Do you want to install the prerequisite packages for the Coral Environmental Board? (y/n)" answer
```

Comment:
wonder if instead of this we can do a treehouses coralenv install,
so we strait say
you can do `apt install ...` or type `treehouses coralenv install`
where on the end of it we tell them to reboot

Reply:
Sounds logical, will add a TODO tag